pallets/loans/src/functions.rs

```diff
@@ -253,7 +254,7 @@ impl<T: Config> Pallet<T> {
 				let new_borrowed_amount = loan_info
 					.borrowed_amount
 					.checked_add(&amount)
-					.ok_or(Error::<T>::AddAmountOverflow)?;
+					.ok_or(DispatchError::Arithmetic(ArithmeticError::Overflow))?;
```

Comment:
I think it's a good idea to use the standard error types for this. Should we use this everywhere, i.e. also in the pools pallet? (doesn't need to be in this PR, we can just create a ticket for this)

Reply:
I think we can proactively do it

Reply:
I didn't realize ArithmeticError existed. We should switch to it :+1: